packages/core/strapi/lib/commands/transfer/import.js

```diff
@@ -26,8 +26,11 @@ module.exports = async (opts) => {
    * From strapi backup file
    */
   const sourceOptions = {
-    backupFilePath: filename,
+    file: { path: filename },
+    compression: { enabled: opts.decompress },
```

Comment:
I think we should still check file extensions, otherwise the user is going to have to know and understand their backup file uses compression and encryption, and will have to manually tell us here, rather than just running `import --file filename.tar` or `import --file filename.tar.gz` or `import --file filename.tar.gz.enc` and having it always just work.

Reply:
I  agree with this statement

Reply:
I've updated the code to automatically detect file extension & related transforms ops